src/pages/index.js

```diff
@@ -1,127 +1,248 @@
 //      Modules     //
-import '../pages/index.css'
+import "../pages/index.css";
 import Card from "../components/Card.js";
 import { FormValidator, settings } from "../components/FormValidator.js";
```

Comment:
**Could be improved**

You could make a special method `resetValidation` for clearing errors and controlling the submit button.
It could look something like this:

```js

resetValidation() {
      this._toggleButtonState(); <== controlling the submit button ==

      this._inputList.forEach((inputElement) => {
        this._hideError(inputElement) <== clearing errors ==
      });

    }



```

And you can call it in `index.js` when you click on open buttons